INSTALLING.rst

```diff
@@ -91,31 +91,28 @@ Tell CMake to search for packages in the virtual environment first::
 Install prerequisites
 ---------------------
 
-**HOOMD-blue** requires a number of libraries to build.
-
-- Required:
-
-  - C++11 capable compiler (tested with ``gcc`` 4.8, 5.5, 6.4, 7,
-    8, 9, ``clang`` 5, 6, 7, 8)
-  - Python >= 3.5
-  - NumPy >= 1.7
-  - pybind11 >= 2.2
-  - Eigen >= 3.2
-  - CMake >= 3.9
-  - For MPI parallel execution (required when ``ENABLE_MPI=on``):
-
+**HOOMD-blue** requires a number of libraries to build. The flags ``ENABLE_MPI``,
+``ENABLE_GPU``, ``ENABLE_TBB``, and ``BUILD_JIT`` each require additional libraries.
+
+General requirements
+    - C++11 capable compiler (tested with ``gcc`` 4.8, 5.5, 6.4, 7,
+      8, 9, ``clang`` 5, 6, 7, 8)
+    - Python >= 3.5
+    - NumPy >= 1.7
+    - pybind11 >= 2.2
+    - Eigen >= 3.2
+    - CMake >= 3.9
+
+For MPI parallel execution (required when ``ENABLE_MPI=on``):
     - MPI (tested with OpenMPI, MVAPICH)
-    - cereal >= 1.1 (required when ``ENABLE_MPI=on``)
-
-  - For GPU execution (required when ``ENABLE_GPU=on``):
+    - cereal >= 1.1
 
+For GPU execution (required when ``ENABLE_GPU=on``):
     - NVIDIA CUDA Toolkit >= 9.0
 
-    **OR**
-
-    - `AMD ROCm >= 2.9 <https://rocm.github.io/ROCmInstall.html>`_
+    *OR*
 
-      Additional dependencies:
+    - `AMD ROCm >= 2.9 <https://rocm.github.io/ROCmInstall.html>`_ with additional dependencies:
         - HIP [with ``hipcc`` and ``hcc`` as backend]
```

Comment:
In nested lists, there needs to be a blank line before the start *and* the `-` needs to line up with the first character of the parent list text (the backtick above).